features/media-queries-prefers-contrast.yml

```
@@ -0,0 +1,6 @@
+name: prefers-contrast
```

```suggestion
name: prefers-contrast media feature
```

For consistency here too.
Same comment for other files below.